src/SFA.DAS.AdminService.Web/Infrastructure/IRoatpApplicationApiClient.cs

```diff
@@ -10,14 +11,14 @@ public interface IRoatpApplicationApiClient
     {
         Task EvaluateSection(Guid applicationId, int sequenceNo, int sectionNo, bool isSectionComplete, string evaluatedBy);
         Task<RoatpApplicationResponse> GetApplication(Guid Id);
-        Task<List<AssessorService.ApplyTypes.Roatp.Apply>> GetClosedApplications();
-        Task<List<AssessorService.ApplyTypes.Roatp.Apply>> GetClosedFinancialApplications();
-        Task<List<AssessorService.ApplyTypes.Roatp.Apply>> GetFeedbackAddedApplications();
-        Task<List<AssessorService.ApplyTypes.Roatp.Apply>> GetFeedbackAddedFinancialApplications();
-        Task<List<AssessorService.ApplyTypes.Roatp.Apply>> GetOpenApplications();
-        Task<List<AssessorService.ApplyTypes.Roatp.Apply>> GetOpenFinancialApplications();
+        Task<List<RoatpApplicationSummaryItem>> GetClosedApplications();
+        Task<List<RoatpFinancialSummaryItem>> GetClosedFinancialApplications();
+        Task<List<RoatpApplicationSummaryItem>> GetFeedbackAddedApplications();
+        Task<List<RoatpFinancialSummaryItem>> GetFeedbackAddedFinancialApplications();
+        Task<List<RoatpApplicationSummaryItem>> GetOpenApplications();
+        Task<List<RoatpFinancialSummaryItem>> GetOpenFinancialApplications();
         Task ReturnApplication(Guid applicationId, string returnType, string returnedBy);
-        Task ReturnFinancialReview(Guid applicationId, FinancialGrade grade);
+        Task ReturnFinancialReview(Guid applicationId, FinancialReviewDetails financialReviewDetails, string financialReviewStatus);
```

Comment:
As mentioned financialReviewStatus should be set appropriately within the Handler in  the API

Reply:
Done